### PR TITLE
Align all `page-title`s & improve `page-description` s (Closes #1331)

### LINKED ISSUE #1331: Align all `page-title`s & improve `page-description` s
## Feature description

Format all `page-title`s in the same way & improve the `page-description`s, they are often very long and not really helpful. 

## Problem and motivation

Consistency between the pages, nicer overview when working with multiple tabs.

## Is this something you're interested in working on

Yes - PR is coming.

### PR DESCRIPTION
**Pull request by Ein-Tim**
### This PR has to be merged **INTO** the `svengabr/screenshot-archive` branch or into `master` **AFTER** #1329.¹

---

This PR introduces #1331. It aligns the `page-title`s on most of the pages and improves their descriptions (`page-description`).

Closes #1331 

---

¹ This is needed because #1329 adds a new site and I wanted to edit this too, so I used `svengabr/screenshot-archive` as the starting branch.

My change is in: https://github.com/corona-warn-app/cwa-website/pull/1332/commits/df977e7028d2b7368989de368510bb5a7294efa5

### CODE REVIEW

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim Thanks. I will merge after the other R 2.3 PRs!

File: src/pages/en/blog/index.html
Comment (MikeMcC399) [inline src/pages/en/blog/index.html:4]: Typo
"developmen on" =>
"development of"

File: src/pages/en/index.html
Comment (MikeMcC399) [inline src/pages/en/index.html:6]: grammar
"from" => "for"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
A couple of quick suggestions for you.

File: src/pages/en/blog/index.html
Comment (Ein-Tim) [reply src/pages/en/blog/index.html]: Corrected in https://github.com/corona-warn-app/cwa-website/pull/1332/commits/2ab687f10cc2cc171e62681496d7121a087c8d5a

File: src/pages/en/index.html
Comment (Ein-Tim) [reply src/pages/en/index.html]: Corrected in https://github.com/corona-warn-app/cwa-website/pull/1332/commits/2ab687f10cc2cc171e62681496d7121a087c8d5a

**Comment by Ein-Tim** [comment]
Thank you @MikeMcC399, I corrected this in https://github.com/corona-warn-app/cwa-website/pull/1332/commits/2ab687f10cc2cc171e62681496d7121a087c8d5a.

**Comment by Ein-Tim** [comment]
Base branch changed to master since it will be merged into master after #1329.

**Don't** merge the latest changes of `master` into this branch to avoid merge conflicts. `Master` can be merged once #1329 is merged into `master`.

File: src/pages/en/blog/index.html
Comment (MikeMcC399) [reply src/pages/en/blog/index.html]: you changed to development, but you didn't change "on" to "of".

https://dictionary.cambridge.org/dictionary/english/development gives an example of "development" followed by "of"

Comment (Ein-Tim) [reply src/pages/en/blog/index.html]: @MikeMcC399 You're right, changed in https://github.com/corona-warn-app/cwa-website/pull/1332/commits/74ad7df09b7525bc6cf4d827ffc319b91ba2deec.

**Comment by heinezen** [comment]
@Ein-Tim Can you change the target branch after creating the PR (with "Edit" next to the title) or is that limited to CWA orga members? I've set the target branch to `svengabr/screenshot-archiv` now, so that the PR can potentially be merged into https://github.com/corona-warn-app/cwa-website/pull/1329 .

**Comment by heinezen** [comment]
@Ein-Tim Why change it to `master` when we could merge it into `svengabr/screenshot-archive` https://github.com/corona-warn-app/cwa-website/pull/1329 too? Does it need the PR to be merged into master beforehand?

**Comment by Ein-Tim** [comment]
@heinezen 

See the comments from @dsarkar https://github.com/corona-warn-app/cwa-website/pull/1332#issuecomment-855361208 & https://github.com/corona-warn-app/cwa-website/issues/1331#issuecomment-855360807, he wants to merge after the other PR, so directly into `master`.

But you can decide what to do, for me it doesn't matter wether we merge it into `svengabr/screenshot-archiv` or into `master` after #1329.

**Comment by heinezen** [comment]
@Ein-Tim Well, changing the target to `master` can be a bit messy, so I would not change that unless it's necessary (i.e. https://github.com/corona-warn-app/cwa-website/pull/1329 was merged). I'll notify the others and check if we can merge directly into the screenshot branch.

**Pull request merged**
